examples/07_advanced/plot_surface_bids_analysis.py

```diff
@@ -47,28 +47,49 @@
 # From the dataset directory we automatically obtain
 # the FirstLevelModel objects
 # with their subject_id filled from the :term:`BIDS` dataset.
-# Moreover, we obtain for each model
-# a dictionary with run_imgs, events and confounder regressors
-# since in this case a confounds.tsv file is available
-# in the :term:`BIDS` dataset.
+# Along, we also obtain:
+#
+#   - a list with the Nifti image associated with each run
+#
+#   - a list of events read from events.tsv in the :term:`BIDS` dataset
+#
+#   - a list of confounder motion regressors
+#     since in this case a confounds.tsv file is available
+#     in the :term:`BIDS` dataset.
+#
 # To get the first level models we only have to specify the dataset directory
-# and the task_label as specified in the file names.
+# and the ``task_label`` as specified in the file names.
+#
+# .. note::
+#
+#       We are only using a subset of participants from the dataset
+#       to lower the run time of the example.
+#
 from nilearn.glm.first_level import first_level_from_bids
 
-task_label = "languagelocalizer"
 models, run_imgs, events, confounds = first_level_from_bids(
-    data.data_dir,
-    task_label,
+    dataset_path=data.data_dir,
+    task_label="languagelocalizer",
     img_filters=[("desc", "preproc")],
+    sub_labels=["01", "02", "03", "04", "05"],  # comment to run all subjects
     hrf_model="glover + derivative",
-    n_jobs=2,
+    n_jobs=-1,
 )
 
 # %%
 # Project :term:`fMRI` data to the surface and compute GLM and contrasts
 #
 # The projection function simply takes the :term:`fMRI` data and the mesh.
 # Note that those correspond spatially, as they are both in :term:`MNI` space.
+#
+# .. warning::
+#
+#    Note that here we pass ALL the confounds when we fit the model.
+#    In this case we can do this because our regressors only include
+#    the motion realignment parameters.
+#    For most preprocessed BIDS dataset,
+#    you would have to carefully choose which confounds to include.
```

Comment:
How about also mentioning the [`load_confounds`](https://nilearn.github.io/stable/modules/generated/nilearn.interfaces.fmriprep.load_confounds.html#nilearn-interfaces-fmriprep-load-confounds) function?

Reply:
good point

Reply:
added something